game.go

```diff
@@ -496,12 +498,19 @@ func (p *Game) loadIndex(g reflect.Value, index interface{}) (err error) {
 	if scenes := proj.getScenes(); len(scenes) > 0 {
 		p.baseObj.init("", scenes, proj.getSceneIndex())
 	} else {
+		if p.windowWidth_ > proj.Map.Width {
+			p.worldWidth_ = p.windowWidth_
```

Comment:
why change world size?
if window is too big, limit window not world.

Reply:
if window is too big,  
-----------
 yes!

, limit window not world.
-----
ok

Reply:
fixed